mobile-app/app/screens/TransactionAuthorization/PasscodePrompt.tsx

```diff
@@ -206,12 +202,14 @@ export const PasscodePrompt = React.memo((props: PasscodePromptProps): JSX.Eleme
           renderBackdrop={() => (
             <View style={{
               position: 'absolute',
+              borderTopRightRadius: 15,
+              borderTopLeftRadius: 15,
               top: 0,
               right: 0,
               bottom: 0,
               left: 0,
               backgroundColor: 'black',
-              opacity: 0.3
+              opacity: 0.6
```

Comment:
can we replace inline css with class?

Reply:
You mean the entire css for the modal? I thought there's reason behind using it, i am wrong?

Reply:
Yes, Our first priority related to css is use existing classes and if we dont have it then either create one only if we are using that at multiple places or at last use inline